MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiDevicePathLib.h

```diff
@@ -0,0 +1,30 @@
+/** @file MockUefiDevicePathLib.h
+  Google Test mocks for UefiDevicePathLib
+
+  Copyright (c) 2022, Intel Corporation. All rights reserved.

```

Comment:
Did this file come from Intel?

Reply:
UefiDevicePathLib.h comes from Intel so yes

Reply:
Vivian is correct.

UefiDevicePathLib comes from Intel, but this is a derivative of it, because its MockUefiDevicePathLib. 

Our copyright should be below the Intel Copyright.

Reply:
Do we need to add as below to all files under MockUefiDevicePathLib :

Copyright (c) 2022, Intel Corporation. All rights reserved.
Copyright (C) Microsoft Corporation
SPDX-License-Identifier: BSD-2-Clause-Patent